settings.gradle

```diff
@@ -200,3 +202,6 @@ if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
   }
 }
 
+include ":iceberg-kafka-connect:kafka-connect-events"
```

Comment:
Spark/Flink/Hive require that we support multiple versions.  Is the KC api stable enough that we don't have to worry about supporting different major/minor versions?  I see the `3.x` line goes back to 2021, but there are six minor releases.  Just wondering if we should structure the project with versioning in mind from the start.

Reply:
For the most part, the API has been very stable so I was thinking of not doing this to start, it might be overkill.

Reply:
Do we need so many modules? If we're just creating a single Jar in the end I wonder if it is helpful to break it up

Reply:
The main reason I have events as separate module is so that it can be used independently from the sink to read and deserialize events from the control topic. This can be used, for example, to trigger workflows from the control topic rather than having to poll the table metadata. With that, the plan was to have 2 modules (events, core) plus a runtime module.